variables.tf

```diff
@@ -132,6 +132,23 @@ variable "configuration" {
   default = null
 }
 
+variable "admin_pass" {
+  type        = string
+  description = "The password for the database administrator. If not specified, an empty string is provided for the password and the user ID cannot be used. In this case, more users must be specified in a user block. The user password must be in the range of 10-32 characters."
+  default     = null
+}
+
+variable "users" {
+  type = list(object({
+    name     = string
+    password = string # pragma: allowlist secret
+    type     = string # "type" is required to generate the connection string for the outputs.
+    role     = optional(string)
+  }))
+  default     = []
+  description = "A list of users that you want to create on the database. Multiple blocks are allowed. The user password must be in the range of 10-32 characters."
```

Comment:
sensitive = true

Reply:
I would put a warning here to indicate that in most case using IAM service credentials (via the service_credential_names variable) is sufficient to control access to the instance. This blocks creates native postgres database users. Perhaps also point to https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-user-management&interface=ui